Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -1426,18 +1427,20 @@ def calculate_next_request_time(delay_request_counter: int) -> float:
     return next_request_time.timestamp()
 
 
-def set_retry_mechanism_arguments(next_request_time: float = 0.0, delay_request_counter: int = 1):
+def set_retry_mechanism_arguments(context: dict, next_request_time: float = 0.0, delay_request_counter: int = 1):
     """
         Sets the next_request_time in the integration context.
         This is an implication of the Moderate Retry Mechanism for the Oproxy requests.
     """
+    context = context if context else {}
```

Comment:
no need to commit if except for this you're ready, but:
```suggestion
    context = context or {}
```